### PR TITLE
SYCL range stuff

### PR DESCRIPTION
**Pull request by jeffhammond**

### CODE REVIEW

File: Cxx11/nstream-sycl.cc [hunk 1]
```diff
@@ -90,7 +92,9 @@ void run(sycl::queue & q, int iterations, size_t length, size_t block_size)
 #if PREBUILD_KERNEL
     auto ctx = q.get_context();
     sycl::program kernel(ctx);
```
Comment (keryell) [inline Cxx11/nstream-sycl.cc:94]: `sycl::program kernel{ctx};`

File: Cxx11/stencil-sycl-usm.cc [hunk 2]
```diff
@@ -105,19 +105,13 @@ void run(sycl::queue & q, int iterations, size_t n, size_t tile_size, bool star,
   // Allocate space and perform the computation
   //////////////////////////////////////////////////////////////////////
 
-  auto ctx = q.get_context();
-
   double stencil_time(0);
```
Comment (keryell) [inline Cxx11/stencil-sycl-usm.cc:108]: `double stencil_time{0};` or even `double stencil_time{};

File: Cxx11/stencil-sycl-usm.cc [hunk 3]
```diff
@@ -273,7 +267,7 @@ int main(int argc, char * argv[])
   //////////////////////////////////////////////////////////////////////
 
   try {
-    sycl::queue q(sycl::host_selector{});
+    sycl::queue q(sycl::host_selector{}, sycl::property::queue::in_order{});
```
Comment (keryell) [inline Cxx11/stencil-sycl-usm.cc:270]: `sycl::queue q{sycl::host_selector{}, sycl::property::queue::in_order{}};`

File: Cxx11/stencil-sycl-usm.cc [hunk 4]
```diff
@@ -290,7 +284,7 @@ int main(int argc, char * argv[])
   }
 
   try {
-    sycl::queue q(sycl::cpu_selector{});
+    sycl::queue q(sycl::cpu_selector{}, sycl::property::queue::in_order{});
```
Comment (keryell) [inline Cxx11/stencil-sycl-usm.cc:287]: `sycl::queue q{sycl::cpu_selector{}, sycl::property::queue::in_order{}};`

File: Cxx11/stencil-sycl-usm.cc [hunk 5]
```diff
@@ -307,7 +301,7 @@ int main(int argc, char * argv[])
   }
 
   try {
-    sycl::queue q(sycl::gpu_selector{});
+    sycl::queue q(sycl::gpu_selector{}, sycl::property::queue::in_order{});
```
Comment (keryell) [inline Cxx11/stencil-sycl-usm.cc:304]: `sycl::queue q{sycl::gpu_selector{}, sycl::property::queue::in_order{}};`

**Review by keryell — COMMENTED** [review]
To be coherent with the neighborhood...

File: Cxx11/stencil-sycl-usm.cc (hunk 2 above)
Comment (jeffhammond) [reply Cxx11/stencil-sycl-usm.cc:108]: yeah, the timer stuff is a mess.  it is not broken just not consistent.  it just isn't important enough to me right now to fix.

File: Cxx11/stencil-sycl-usm.cc (hunk 3 above)
Comment (jeffhammond) [reply Cxx11/stencil-sycl-usm.cc:270]: oops.  will fix.

File: Cxx11/stencil-sycl-usm.cc (hunk 5 above)
Comment (jeffhammond) [reply Cxx11/stencil-sycl-usm.cc:304]: i just remembered i made this change on a different branch and need to redo it on the right one.

**Pull request merged**